.github/workflows/tests.yml

```
@@ -28,20 +28,15 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Install cylc-flow
+      - name: Install deps
         run: |
-          pip install 'cylc-flow[all] @ git+https://github.com/cylc/cylc-flow@master'
+          pip install 'cylc-flow @ git+https://github.com/cylc/cylc-flow@master'
```

I think the tests are failing because empy is not getting installed

```suggestion
          pip install 'cylc-flow[all] @ git+https://github.com/cylc/cylc-flow@master'
```

---

Either that or we add empy to the "tests" requirements for cylc-rose

---

```suggestion
          pip install 'cylc-flow[empy] @ git+https://github.com/cylc/cylc-flow@master'
```

---

Lets give it a go.